### ISSUE TITLE
Increase base build to 2x daily

### ISSUE THREAD

**Issue by kramfs** [ublue-os/base#29]
This is a request if the daily build frequency can be updated to at least more than once per day. I'm basing on this OCI image and some upstream updates are available during my day (UTC +8) but I have to wait until the next day for an updated image to become available. Running the build 2x day would pretty much cover most TZs.

**Comment by ahgencer** [comment]
Unless the upstream base image (`ghcr.io/cgwalters/fedora-silverblue`) from [this](https://github.com/cgwalters/sync-fedora-ostree-containers) repo also starts rebuilding twice daily, kicking off a new workflow run more frequently would not have any affect on the package versions that end up in the image.

**Comment by kramfs** [comment]
The upstream [repo](https://quay.io/repository/fedora-ostree-desktops/silverblue?tab=tags&tag=37) changed and there are indications the build is happening more than once per day hence the request. Just a minor request, it wouldn't put too much strain on the pipeline to increase the build cadence...

**Comment by ahgencer** [comment]
Oh! You're right, looks like I missed 5229061.

Just a quick question: Where exactly are the new base images coming from? I thought the official images were waiting on [this](https://pagure.io/releng/pull-request/11180) PR getting merged. The official Fedora CoreOS image is at `quay.io/fedora/fedora-coreos`; that PR links to [this](https://pagure.io/releng/issue/11047) issue which suggests extending that scheme with `quay.io/fedora/fedora-silverblue` for example.

Going up to the [organization level](https://quay.io/organization/fedora-ostree-desktops) for `fedora-ostree-desktops`, there are also community-built "Silverblue-like" variants such as `sericea` and `vauxite`. So I'm wondering, is `fedora-ostree-desktops` another community-maintained collection of images, or is it an extension of the official `fedora` organization?

Also, where exactly are the sources that the `fedora-ostree-desktops` images build from? I can't see any upstream repository linked on the Quay.io page.

**Comment by ahgencer** [comment]
Ok, after searching around for a while, the new variants seem to be defined in [this](https://pagure.io/workstation-ostree-config) repo on Pagure. The `fedora-ostree-desktops` organization Quay.io is not "oficially" part of the Fedora project, but it is maintained by Timothée Ravier, a Red Hat CoreOS engineer. However, I couldn't find where the container images are being encapsulated and pushed onto the registry.

> So I'm wondering, is `fedora-ostree-desktops` another community-maintained collection of images, or is it an extension of the official `fedora` organization?

So in a sense, it's both. But until Fedora starts releasing "official" images at `quay.io/fedora/fedora-silverblue`, `fedora-ostree-desktops` also seems to be a temporary solution.

- https://tim.siosm.fr/blog/2023/01/20/introducing-kinoite-nightly-beta/

**Comment by ahgencer** [comment]
> However, I couldn't find where the container images are being encapsulated and pushed onto the registry.

Curiosity got the better of me, but I finally stumbled upon [this](https://gitlab.com/fedora/ostree/ci-test) repository that builds the images using GitLab's CI. Taking a look at the [job schedule](https://gitlab.com/fedora/ostree/ci-test/-/pipeline_schedules), the new base image is also pushed once per day to Quay.io, so it isn't scheduled to be updated any more frequently than the old one.

**Comment by castrojo** [comment]
Yeah I think what we ultimately need is a way to just monitor the upstream image, and when it changes just build on the spot, then we wouldn't have to care what the upstream cadence is, we'd just be as up to date as possible. 

I don't know how we would implement this though, it's above my skill level but let's leave this open!

**Comment by castrojo** [comment]
Maybe something like this? https://github.com/giggio/docker-image-update-checker

**Comment by ahgencer** [comment]
Perhaps we should merge this issue with #4 then? It seems there isn't much we can do unless the upstream cadence changes.
